From 245e4f0da830830c6ca9b59ce9eb2f7d9ba6e0a5 Mon Sep 17 00:00:00 2001
From: Dongdong Liu <liudongdong3@huawei.com>
Date: Tue, 11 Oct 2022 19:18:43 +0800
Subject: doc: document device dump in procinfo guide

[ upstream commit dd2658f8d0e4725f579ff098f1ee159d897d6abc ]

The --show-port-private option was not documented.

Fixes: bb947a7264da ("app/procinfo: dump device private info")

Signed-off-by: Dongdong Liu <liudongdong3@huawei.com>
Signed-off-by: David Marchand <david.marchand@redhat.com>
---
 doc/guides/tools/proc_info.rst | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/doc/guides/tools/proc_info.rst b/doc/guides/tools/proc_info.rst
index e4f0c83f1b..5dd6f9ecae 100644
--- a/doc/guides/tools/proc_info.rst
+++ b/doc/guides/tools/proc_info.rst
@@ -19,7 +19,8 @@ The application has a number of command line options:
 
    ./<build_dir>/app/dpdk-proc-info -- -m | [-p PORTMASK] [--stats | --xstats |
    --stats-reset | --xstats-reset] [ --show-port | --show-tm | --show-crypto |
-   --show-ring[=name] | --show-mempool[=name] | --iter-mempool=name ]
+   --show-ring[=name] | --show-mempool[=name] | --iter-mempool=name |
+   --show-port-private ]
 
 Parameters
 ~~~~~~~~~~
@@ -69,6 +70,9 @@ mempool. For invalid or no mempool name, whole list is dump.
 The iter-mempool parameter iterates and displays mempool elements specified
 by name. For invalid or no mempool name no elements are displayed.
 
+**--show-port-private**
+The show-port-private parameter displays ports private information.
+
 Limitations
 -----------
 
-- 
2.23.0

